### PR TITLE
feat: Add relationship between Invoice and Suppliers

### PR DESCRIPTION
**Pull request by WebertySilveira**

### CODE REVIEW

File: app/Models/Invoice.php [hunk 1]
```diff
@@ -34,4 +36,9 @@ public function merchant(): BelongsTo
     {
         return $this->belongsTo(Merchant::class, 'merchant_id');
     }
+
+    public function supllier(): hasOne
```
Comment (kayobruno) [inline app/Models/Invoice.php:40]: What do you think about create a test to validate this relationship?

**Review by kayobruno — CHANGES_REQUESTED** [review]

**Review by kayobruno — COMMENTED** [review]
Can you use the Conventional Commits Pattern please?
below i share the link about this Pattern:
https://medium.com/linkapi-solutions/conventional-commits-pattern-3778d1a1e657

File: app/Models/Invoice.php (hunk 1 above)
Comment (WebertySilveira) [reply app/Models/Invoice.php]: Extremely necessary. I adjusted the fields that were incorrect for an invoice to belong to a supplier, and increased the invoice creation test to predict the supplier

**Review by kayobruno — APPROVED** [review]
LGTM!

**Pull request merged**